web/src/message_list.ts

```diff
@@ -539,22 +539,17 @@ export class MessageList {
 
     show_edit_topic_on_recipient_row($recipient_row: JQuery, $form: JQuery): void {
         $recipient_row.find(".topic_edit_form").append($form);
-        $recipient_row.find(".on_hover_topic_edit").hide();
         $recipient_row.find(".stream_topic").hide();
         $recipient_row.find(".topic_edit").show();
-        $recipient_row.find(".always_visible_topic_edit").hide();
-        $recipient_row.find(".on_hover_topic_resolve").hide();
-        $recipient_row.find(".on_hover_topic_unresolve").hide();
+        $recipient_row.find(".recipient_bar_control").hide();
```

Comment:
I'd prefer going with our preferred hyphenated classes here, since this is being introduced anew.

Reply:
Fixed.